source/opentelemetry-collector-contrib/examples/tracing/README.md

```diff
@@ -0,0 +1,49 @@
+# OpenTelemetry Collector Demo
+
+This demo is a sample Python app to build the collector and exercise its tracing functionality. To build and run the demo, we will need to deploy the Python app and the collector and send traces to Azure Monitor. 
+
+## Deploying the Python Application
+#### 1. Create Docker image for Python app
+`docker build -f Dockerfile.python . -t <name:tag>` <br/>
+`docker push <name:tag>` <br/>
+
+#### 2. Delete previous deployments
+`kubectl delete -f python-deployment.yaml`<br/>
+
+#### 3. Check deployment and related pods are deleted
+`kubectl get deployments`<br/>
+`kubectl get pods`<br/>
+
+#### 4. Deploy
+`kubectl apply -f python-deployment.yaml`<br/>
+
+Once that has been completed, it is time to deploy the collector.
+
+## Deploying the Collector
+
```

Comment:
Missing the instructions to build collector binaries before building docker image.